hw/ip/edn/data/edn_testplan.hjson

```diff
@@ -23,32 +24,36 @@
       name: firmware
       desc: '''
             Verify SW_CMD_REQ/SW_CMD_STS registers/bits behave as predicted.
-            Verify RESEED/GENERATE software cmds work with 0 and 12 32-bit words of additional data.
+            Verify software mode behaves as predicted.
+            Verify INSTANTIATE/GENERATE software cmds.
             Verify cmd_fifo_reset bit causes fifos to reset.
-            Verify boot_req_dis bit disables boot_req mode.
-            Verify registers at End-Of-Test.
             '''
       milestone: V2
-      tests: []
+      tests: ["edn_genbits"]
     }
     {
-      name: csrng_cmd
+      name: csrng_commands
       desc: '''
-            Verify endpoint agent reqs generate csrng commands for both fifo full/empty conditions.
             Verify when no/some/all endpoints requesting (test arbiter).
-            Verify auto-request mode (RESEED_CMD/GENERATE_CMD registers) behaves as predicted.
-            Verify max_num_reqs_between_reseeds in auto-generate mode.
-            Verify boot-time request mode behaves as predicted.
+            Verify boot request mode behaves as predicted.
+            Verify BOOT_INS_CMD/BOOT_GEN_CMD registers.
+            Verify auto request mode behaves as predicted.
+            Verify RESEED_CMD/GENERATE_CMD/MAX_NUM_REQS_BETWEEN_RESEEDS registers.
             Verify SUM_STS register bits behave as predicted.
+            Verify all csrng commands (clen = 0-12, sw_mode, boot/auto_req_mode).
+            // TODO: Implemented in PR#11232, to be merged
+            Verify with ready randomly asserting/deasserting
             '''
       milestone: V2
-      tests: []
+      tests: ["edn_genbits"]
     }
     {
       name: genbits
       desc: '''
             Verify genbits input is transferred to endpoint(s) as predicted.
             Verify fips bit(s) are properly transferred to endpoint.
+            // TODO: Implement below in vseq
+            Endpoints request non-multiple of ENDPOINT_BUS_SIZE/GENBITS_BUS_SIZE.
```

Comment:
should we address this before sign off V2?

Reply:
Maybe. I'll look into it but I'd rather not have it hold up this PR. WDYT?